core/core.go

```diff
@@ -130,6 +130,13 @@ type Config struct {
 	// enforcing a limit on maximum expiration time for incoming orders and remove
 	// any orders with an expiration time too far in the future.
 	MaxOrdersInStorage int `envvar:"MAX_ORDERS_IN_STORAGE" default:"100000"`
+	// CustomOrderFilter is a JSON-schema which will be used for validating
+	// incoming orders. If provided, Mesh will only receive orders from other
+	// peers in the network with the same filter.
+	//
+	// TODO(albrow): Link to more documentation about JSON-schemas and how this
+	// filter works.
+	CustomOrderFilter string `envvar:"CUSTOM_ORDER_FILTER" default:""`
```

Comment:
Can we make the default `"{}"` instead of an empty string? The former is actually what the default schema looks like and elucidates to the reader more clearly the form in which the schema is expected to be received (a stringified JSON object).

Reply:
Sure. Done.